modules/seqexec/web/shared/src/main/scala/seqexec/web/model/boopickle/ModelBooPicklers.scala

```diff
@@ -26,17 +29,38 @@ import seqexec.model.events._
     "org.wartremover.warts.OptionPartial"
   ))
 trait ModelBooPicklers extends GemModelBooPicklers {
-  //**********************
-  // IMPORTANT The order of the picklers is very relevant to the generated size
-  // add them with care
-  //**********************
-  implicit val instrumentPickler = generatePickler[Instrument]
+  def valuesMap[F[_]: Functor: Foldable, A, B](c: F[A], f: A => B): Map[B, A] =
```

Comment:
@tpolecat, does this make sense? Do you know why I can’t do just `valueMap(List, _.ordinal)` when calling it?

Instead I have to declare explicitly the arg type